### DISCUSSION TITLE
Bootlooping fabric?

### DISCUSSION THREAD

**Discussion by Martin-963** [FabricMC/fabric-api#2735] [Player support] [answered]
Okay so hi, i recently updated to Fabric 1.19.3 from an instance on MultiMC containing minecraft 1.19.2 and fabric of its version. I replaced the Minecraft's version via MultiMC, removed Fabric completely and reinstalled it. MultiMC downloaded new 1.19.3 libraries and it had no fabric. So i tried reinstalling fabric again and it started bootlooping. I even tried entire NEW INSTANCE with same specs (mods, texture packs, shaders, fabric installer and version). Any ideas on what's happening??

**video**: https://user-images.githubusercontent.com/82881733/206782363-32f932d8-406f-48dc-b32d-1866d5c5ed61.mp4
**logs**: https://paste.ee/p/jHTAz
**mods list**: ![image](https://user-images.githubusercontent.com/82881733/206782188-4a4d232f-d562-478a-a239-a84e4438f47a.png)

**Comment by Juuxel** [comment]
The "boot loops" with resource loading usually happen because of errors during resource reloading. They should be in the game log (`logs/latest.log`).

Edit: oh, I didn't notice the log was attached already 🤦

**Comment by warjort** [comment, marked as answer]
````
Caused by: java.lang.NoSuchFieldError: field_11142
	at dev.lambdaurora.lambdynlights.api.item.ItemLightSource.fromJson(ItemLightSource.java:99) ~[lambdynamiclights-2.1.2+1.19.jar:?]
````
Looks like an issue with lambdynamiclights not being compatible with 1.19.3?

**Reply by Martin-963** [reply]
Thank you so much!
